products/dns/api.yaml

```diff
@@ -89,39 +89,44 @@ objects:
         description: |
           Unique numeric identifier for the resource; defined by the server.
         output: true
-      - !ruby/object:Api::Type::Integer
-        name: 'quota.managedZones'
-        description: Maximum allowed number of managed zones in the project.
-        output: true
-      - !ruby/object:Api::Type::Integer
-        name: 'quota.resourceRecordsPerRrset'
-        description: |
-          Maximum allowed number of ResourceRecords per ResourceRecordSet.
-        output: true
-      - !ruby/object:Api::Type::Integer
-        name: 'quota.rrsetAdditionsPerChange'
-        description: |
-          Maximum allowed number of ResourceRecordSets to add per
-          ChangesCreateRequest.
-        output: true
-      - !ruby/object:Api::Type::Integer
-        name: 'quota.rrsetDeletionsPerChange'
-        description: |
-          Maximum allowed number of ResourceRecordSets to delete per
-          ChangesCreateRequest.
-        output: true
-      - !ruby/object:Api::Type::Integer
-        name: 'quota.rrsetsPerManagedZone'
-        description: |
-          Maximum allowed number of ResourceRecordSets per zone in the
-          project.
-        output: true
-      - !ruby/object:Api::Type::Integer
-        name: 'quota.totalRrdataSizePerChange'
-        description: |
-          Maximum allowed size for total rrdata in one ChangesCreateRequest
-          in bytes.
+      - !ruby/object:Api::Type::NestedObject
```

Comment:
You need to add an entry to `puppet.yaml` and `chef.yaml` to warn about this change.

Reply:
What did you mean by this?

Reply:
Updated.

(Chef DNS did not previously have a changelog).